examples/counterexamples/README.md

```diff
@@ -0,0 +1,8 @@
+Every CSDL file in this folder represents a test case where schema validation that is meant to fail at a certain line and column because a certain XML schema rule is violated.
+
+Per CSDL file there must be three entries in the [`test.properties`](test.properties) file:
+```
+filename.xml.line=<line number>
+filename.xml.col=<column number>
+filename.xml.rule=<rule name>
```

Comment:
How is `<rule name>` formed? Do I have to insert `dummy` here, have the test tool fail, and then copy its rule name output?

That would be ok, and then please describe it here.